src/main/java/lotto/domain/Money.java

```diff
@@ -0,0 +1,23 @@
+package lotto.domain;
+
+public class Money {
+    private static final long LOTTO_PRICE = 1000;
```

Comment:
돈을 나타내는 숫자의 경우 천 단위를 구분하기 위해 "1_000"와 같이 언더바를 통해 구분할 수 있다.
```suggestion
    private static final long LOTTO_PRICE = 1_000;
```